src/content/en/2019/third-parties.md

```diff
@@ -0,0 +1,167 @@
+
+### Simplify Development
+
+Above all third-party resources are used to simplify the web development experience. Even previous usage patterns could arguably fall into this pattern as well. Whether analyzing user behavior, communicating with advertisers, or personalizing the user experience, third-party resources are used to make first-party development easier.
+
+Hosting providers are the most extreme example of this pattern. Some of these providers even enable anyone on Earth to become a site owner with no technical expertise necessary. They provide hosting of assets, tools to build sites without coding experience, and domain registration services.
+
+The remainder of third-party providers also tend to fall into this usage pattern. Whether it’s hosting of a utility library such as jQuery for usage by front-end developers cached on Cloudflare’s edge servers or a vast library of common fonts served from a popular Google CDN, third-party content is another way to give the site owner one fewer thing to worry about and, maybe, just maybe make the job of delivering a great experience a little bit easier.
+
+## Repercussions
+
+### Performance
+
+The performance impact of third-party content is neither categorically good or bad. There are good and bad actors across the spectrum and different category types have varying levels of influence.
+
+The good: shared third-party font and stylesheet utilities are, on average, delivered more efficiently than their first-party counterparts.
+
+Utilities, CDNs, and Content categories are the brightest spots on the third-party performance landscape. They offer optimized versions of the same sort of content that would otherwise be served from first-party sources. Google Fonts and Typekit serve optimized fonts that are smaller on average than first-party fonts, Cloudflare CDN serves a minified version of open source libraries that might be accidentally served in development mode by some site owners, Google Maps SDK efficiently delivers complex maps that might otherwise be naively shipped as large images.
+
+The bad: a very small set of entities represent a very large chunk of JavaScript execution time carrying out narrow set of functionality on pages.
+
+Ads, social, hosting, and certain analytics providers represent the largest negative impact on web performance. While hosting providers deliver a majority of a site’s content and will understandably have a larger performance impact than other third-party categories, they also serve almost entirely static sites that demand very little JavaScript in most cases that should not justify the volume of script execution time. The other categories hurting performance though have even less of an excuse. They fill very narrow roles on each page they appear on and yet quickly take over a majority of resources.
```

Comment:
Can you give a few examples like you did for the Utilities, CDNs, etc... above?

Reply:
yeah I'll try to walk the line a bit, specifically calling out individual bad actors seemed trickier than praising some best practices :)

Reply:
Yeah very true. Since that may be difficult, you could also just give examples of what types of _narrow roles_ you're talking about. Letting readers fill in the blank themselves

Reply:
Tried to give an example while sticking to talking about the impact of the category as a whole :) hope it addresses what you were getting at

Reply:
Perfect :ok_hand: